README.md

```diff
@@ -83,6 +83,12 @@ if (result.Error != BluetoothError.Success)
 serviceProvider = result.ServiceProvider;
 ```
 
+Optionally the `Create` function can be supplied with an extra argument specifying the Bluetooth device.
```

Comment:
you will have to adjust this one as well as the default null is not an option anymore